kratos/utilities/cpp_tests_utilities.h

```diff
@@ -0,0 +1,73 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_CPP_TESTS_UTILITIES)
+#define KRATOS_CPP_TESTS_UTILITIES
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+/**
+ * @namespace CppTestsUtilities
+ * @ingroup KratosCore
+ * @brief This namespace includes utilities for simplifying the deploy of C++ tests
+ * @author Vicente Mataix Ferrandiz
+ */
+namespace CppTestsUtilities
```

Comment:
if this should go in, then please describe better the geometry that is created

Reply:
Done